docs/kubernetes-deployment.md

```diff
@@ -128,5 +128,29 @@ ports:
 ...
 ```
 
+### Resources allocations
+
+It is recommended to always set the kubernetes [resource](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits) usage such as cpu and memory.
+Kindly note that it is best not to use `resources.limits.cpu` limits but use `resources.requests.cpu` configuration instead.
```

Comment:
I would remove this or change the copy.

In k8s setting the limit is a best practice